cfpb/cfgov-refresh
# Remove IE10 JavaScript support

Gives IE10 the `no-js` treatment, like IE9 before it.

## Removals

- Removes `dataset` polyfill for the video player.

## Changes

- Requires `Map` support for the JS state of the site to work.
- Minor code comment tweaks.

## Testing

1. Pull branch and run `./frontend.sh`
2. Open local site in IE10. http://localhost:8000/prepaid-rule should launch the video in a new tab and play the video in-browser in a modern browser.
3. http://localhost:8000/owning-a-home/ and charts on http://localhost:8000/data-research/mortgage-performance-trends/mortgages-30-89-days-delinquent/ should not work. 

## Note
We use `pointer-event` in CF `a-select` and depend on modernizr to add the `no-csspointerevents` class globally (https://github.com/cfpb/capital-framework/blob/canary/src/cf-forms/src/atoms/select.less#L79 and https://github.com/cfpb/cfgov-refresh/blob/master/gulp/tasks/scripts.js#L58). I've left this structure in place since it's for the CSS.
